src/test/java/redis/clients/jedis/misc/AutomaticFailoverTest.java

```diff
@@ -34,9 +26,9 @@ public class AutomaticFailoverTest {
 
   private static final Logger log = LoggerFactory.getLogger(AutomaticFailoverTest.class);
 
-  private final HostAndPort hostPort_1 = new HostAndPort(HostAndPorts.getRedisServers().get(0).getHost(), 6378);
-  private final HostAndPort hostPort_1_2 = HostAndPorts.getRedisServers().get(0);
-  private final HostAndPort hostPort_2 = HostAndPorts.getRedisServers().get(7);
+  private final HostAndPort hostPortWithFailure = new HostAndPort(HostAndPorts.getRedisEndpoint("standalone0").getHost(), 6378);
+  private final EndpointConfig workingEndpointWithPriority1 = HostAndPorts.getRedisEndpoint("standalone0");
```

Comment:
`endpointForAuthFailure`?

It can be worked on, but here it is being used for failure (auth).